templates/docs/examples/patterns/navigation/sliding.html

```diff
@@ -127,9 +127,8 @@
   body { margin: 0; }
 </style>
 
-<script>
-  {% include 'docs/examples/patterns/navigation/_script-sliding.js' %}
-
-  initNavDropdowns()
+<script type="module">
+  import {initNavDropdowns} from '/build/js/out/sliding-navigation.js';
+  initNavDropdowns();
 </script>
```

Comment:
@bartaz @pedoch here's the init as a module

Reply:
Great! Now, is this part of npm package?

I guess the next step would be to add this to npm and make sure it's exported so you can do `import {initNavDropdowns} from "vanilla-framework"` in external project.

Reply:
I just pushed some changes. It breaks the demo because the new `/js` folder is not exported under `static` url but It think it would work like `import {initNavDropdowns} from "vanilla-framework/js"`

I was thinking of putting the `index.js` at the root of the folder too but idk if that would work